content/news/XXXX-XX-XX-russians-war.md

```diff
@@ -0,0 +1,18 @@
+title: Russian's war on Ukraine
+authors: Daniel Schürmann
+tags: russian ukraine war
+status: draft
+comments: yes
+
+Since 2022-02-24 we receive horrible news from Ukaine, which have become even worse since then. As an organization committed to freedom we feel the urge speak up and show solidarity for Ukraine.
```

Comment:
Don't exaggerate, keep it and simple and neutral: "Since 2022-02-24 we are receiving concerning news from Ukraine." (There was also a typo "Ukaine")

Reply:
I see news where whole families including childs are murdered or a group of young 150 solders has been killed. That is horrible. Calling this concerning feels misplaced. The situation was concerning before 2022-02-24.